### PR TITLE
15 Add bringup package with lidar

### PR DESCRIPTION
**Pull request by griswaldbrooks**
Depends on #13 

`ros2 launch c3pzero_bringup bringup.launch.py`
`ros2 launch c3pzero_bringup bringup.launch.py rviz:=True`

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
   -v $XDG_RUNTIME_DIR:$XDG_RUNTIME_DIR \
+  -v "/etc/group:/etc/group:ro" \
+  -v "/etc/passwd:/etc/passwd:ro" \
+  -v "/etc/shadow:/etc/shadow:ro" \
+  --group-add=dialout \
   -e DISPLAY=$DISPLAY \
   -e QT_X11_NO_MITSHM=1 \
   -e ROS_DOMAIN_ID=89 \
   --privileged \
+  --device=/dev/ttyUSB0 \
```
Comment (MarqRazz) [inline run.sh:15]: I believe the Roboclaw also requires `/dev/ttyACM0`. Do we need to add that here too?

**Review by MarqRazz — COMMENTED** [review]
I will test this with hardware next chance I get.

File: run.sh (hunk 1 above)
Comment (griswaldbrooks) [reply run.sh:15]: Yes, good point

Comment (griswaldbrooks) [reply run.sh:15]: Done

**Pull request merged**